style.css

```diff
@@ -75,3 +82,191 @@ main {
 .icon-link {
   text-decoration: none;
 }
+
+.card-container {
+  padding: 24px;
+  display: grid;
+  justify-content: center;
+}
+
+.card-item {
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  margin-top: 88px;
+  margin-left: 24px;
+  margin-right: 24px;
+  padding: 16px;
+  border-radius: 16px;
+}
+
+.card-heading {
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  padding: 12px 0 0 0;
+}
+
+.canopy {
+  color: #344563;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  padding: 20px 0 0 0;
+}
+
+.gray-matter {
+  font-weight: 600;
+  color: #7a869a;
+  font-size: 13px;
+  padding: 20px 0 0 0;
+}
+
+.card-paragraph {
+  color: 344563;
+  font-size: 15px;
+  line-height: 24px;
+  font-weight: 400;
+  padding: 20px 0 0 0;
+}
+
+.card-list {
+  display: flex;
+  gap: 8px;
+  list-style: none;
+  padding: 12px 0 0 0;
+}
+
+.card-list-items {
+  padding: 4px 12px 4px 12px;
+  border-radius: 20px;
+  background-color: #ebebff;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  color: #6070ff;
+}
+
+.card-button {
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  padding: 12px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  color: #396df2;
+  align-self: flex-start;
+  margin-top: 24px;
+  background-color: #fff;
+}

```

Comment:
Your button looks good, however, you have not implemented the button: hover, button: pressed and button: active styles which are core requirements. Kindly do so to enhance your user accessibility.

Refer to the screenshot below
![image](https://user-images.githubusercontent.com/81932478/164516443-3d925f9f-af78-4a65-93ee-68d3438b40c5.png)

Or you can check the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)